### PR TITLE
Created register page

### PR DESCRIPTION
**Pull request by edenmaor**
created a register html page
added a URL path for register
referred from the login page to the register page

#42 closed.

### CODE REVIEW

**Review by shayakrach — CHANGES_REQUESTED** [review]
Same comments as #47 PR (about the register page).

**Review by AshkenYariv — APPROVED** [review]
The code is approved! Looks like it depends on the create login page but is not a problem in my opinion, @yuvalk please approve the login page before this.

**Comment by edenmaor** [comment]
> The code is approved! Looks like it depends on the create login page but is not a problem in my opinion, @yuvalk please approve the login page before this.

@AshkenYariv Hi yariv it indeed depends on the login page so that we can refer to the register page from inside the login in case someone isn't logged in yet and need to register.

File: templates/register.html [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+    </div>
+
+    <div id="menu1" class="tab-pane fade">
+
+<form action="#">
+
+  <div class="form-group">
+    <label for="UserName">Full Name</label>
+    <input type="text" class="form-control" id="email">
```
Comment (AshkenYariv) [inline templates/register.html]: I think you should change the id

File: templates/register.html [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+
+  <div class="tab-content">
+    <div id="home" class="tab-pane fade in active">
+
+<form action="#">
+
+  <div class="form-group">
+    <label for="UserName">Full Name</label>
+    <input type="text" class="form-control" id="email">
```
Comment (AshkenYariv) [inline templates/register.html]: I think you should change the id here

**Review by AshkenYariv — COMMENTED** [review]
@edenmaor I have not too strong on HTML but I think you need in the forms to add a name value to every section we are asking a user to input so we can use that info registering a user into the DB. What do you think?

File: templates/register.html (hunk 2 above)
Comment (edenmaor) [reply templates/register.html]: fixed

**Review by assafg95 — APPROVED** [review]

**Review by AshkenYariv — APPROVED** [review]

**Comment by AshkenYariv** [comment]
Approved

**Comment by yuvalk** [comment]
plz squash commits

**Comment by edenmaor** [comment]
@yuvalk commit was squashed and conflicts resolved. can now approve and merge PR

**Review by shayakrach — CHANGES_REQUESTED** [review]
Please fix login.html style.

**Comment by edenmaor** [comment]
> Please login.html style.

not sure i understand the change suggestion? @shayakrach

**Comment by shayakrach** [comment]
@edenmaor it is better to fetch&rebase instead of fetch&merge.

You should squash "Merge branch 'master' into registerPageFeat" commit.

**Comment by edenmaor** [comment]
> @edenmaor it is better to fetch&rebase instead of fetch&merge.
> 
> You should squash "Merge branch 'master' into registerPageFeat" commit.
@shayakrach 

its not possible because i have conflicts so when i will do rebase and after that resolved conflicts i automatically needs to handle the conflicts and merge after that.

**Comment by shayakrach** [comment]
> > Please login.html style.
> 
> not sure i understand the change suggestion? @shayakrach


Fix the redundant line spacing and the indentation.

**Comment by edenmaor** [comment]
> @edenmaor it is better to fetch&rebase instead of fetch&merge.
> 
> You should squash "Merge branch 'master' into registerPageFeat" commit.

also there is only one commit :)

**Comment by edenmaor** [comment]
> Please fix login.html style.

fixed.

**Comment by shayakrach** [comment]
> > @edenmaor it is better to fetch&rebase instead of fetch&merge.
> > You should squash "Merge branch 'master' into registerPageFeat" commit.
> > @shayakrach
> 
> its not possible because i have conflicts so when i will do rebase and after that resolved conflicts i automatically needs to handle the conflicts and merge after that.

OK

**Review by shayakrach — APPROVED** [review]

**Review by assafg95 — APPROVED** [review]

**Comment by AshkenYariv** [comment]
@yuvalk @edenmaor @ormergi please merge this PR ASAP so we can continue to next stage.

**Review by AshkenYariv — CHANGES_REQUESTED** [review]
Not relevant remove this

**Comment by edenmaor** [comment]
Not relevant, go to PR #58 for the register page.

**Pull request closed without merging**